src/Paprika.Tests/Merkle/HashingTests.cs

```diff
@@ -0,0 +1,48 @@
+using Nethermind.Int256;
+using NUnit.Framework;
+using Paprika.Crypto;
+using Paprika.Data;
+using Paprika.Merkle;
+using Paprika.RLP;
+
+namespace Paprika.Tests.Merkle;
+
+public class HashingTests
+{
+    private static object[] _keysBalancesNoncesHexStrings =
+    {
+        new object[]
+        {
+            Values.Key0, Values.Balance0, Values.Nonce0,
```

Comment:
Would be super useful if you could provide a link to a branch in NethermindEth maybe where you calculate these. Once they are merged, it will be fine but for now, keeping any kind of link would make the life much easier.

Reply:
I've created a branch with the same tests: https://github.com/NethermindEth/nethermind/tree/feature/paprika_merkle_tests.

Reply:
Could you commit this link to the repository? Otherwise it will get lost. Just add it to the tests.

Reply:
I actually did that but I forgot to push the changes. Whoops!